composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-tokenizer": "*",
         "symfony/console": "^2.3 || ^3.0",
         "symfony/event-dispatcher": "^2.1 || ^3.0",
-        "symfony/filesystem": "^2.1 || ^3.0",
+        "symfony/filesystem": "^2.4 || ^3.0",
```

Comment:
On the other side, if we don't need 2.7 there is no need to enforce it, and ^2.4|^3 will work with 2.7 as well.